### ISSUE TITLE
Example with gwt json overlay

### ISSUE THREAD

**Issue by leforthomas** [stoicflame/enunciate#467]
May be I am missing something but I cannot work out how to use the jar file produced for the gwt json overlay in my project. I could only find some code on the wiki page but not how to include it in the project. With GWT there is normally the need to inherit a module. Please let me know if I missed something, otherwise it would be nice to have a bit more documentation on this nice feature.

**Comment by stoicflame** [comment]
Yes, it's just the overlay code, so you'll have to compile the module yourself. 

I'll keep this issue open until more docs are provided. In the meantime, if you'd like to try it out and write down your notes and/or findings, I'd be happy to include it in the documentation.
